app/_includes/md/konnect/generic-sso.md

```diff
@@ -0,0 +1,122 @@
+<!-- used in the Dev Portal generic SSO how to and the Org generic SSO how to -->
+
+## Prerequisites
+
+* {{site.konnect_short_name}} must be added to your IdP as an application
+* Users that need to use SSO are added to your IdP tenant
+* Claims are set up in your IdP
+
+
+## Set up SSO in {{site.konnect_short_name}}
+{% navtabs %}
+{% navtab OIDC%}
+{% if include.desc == "Konnect Org" %}
```

Comment:
For reviewers: This label in the include file pulls the content from the `{% if include.desc == "Konnect Org" %}` to the next `{% endif %}` into _only_ the Konnect org SSO docs, not the Dev Portal SSO docs. This lets us share most of the content and conditionally render content that is specific to one feature and not the other, like the UI instructions to get to the SSO settings in Konnect.

Reply:
Very powerful! Thank you!